client/src/components/MyAccount/AddItemFormSteps/StepThree.jsx

```diff
@@ -31,6 +38,20 @@ const StepThree = ({
     });
   }, [user]);
 
+  useEffect(() => {
+    (async () => {
+      const response = await getUserCard(user.id);
+      setCardDetails({
+        cardHolderName:
```

Comment:
`cardHolderName: !!data.cardHolderName ? data.cardHolderName : ""` 
Do not only check null, it can be some of other falsy values as well.